tests/pytests/data/isds/kaguyami_isd.json

```diff
@@ -1,51 +1,282 @@
+  },
+  "instrument_pointing": {
+    "time_dependent_frames": [
+      -131000,
+      1
+    ],
+    "ck_table_start_time": 274867895.6899549,
+    "ck_table_end_time": 274867908.1699299,
+    "ck_table_original_size": 5,
+    "ephemeris_times": [
+      274867895.6899549,
+      274867898.8099486,
+      274867901.92994237,
+      274867905.0499362,
+      274867908.1699299
+    ],
+    "quaternions": [
+      [
+        -0.211189167087694,
+        -0.591758548842394,
+        0.07283594721177017,
+        -0.7745423683508311
+      ],
+      [
+        -0.21125161922443972,
+        -0.5928173480877278,
+        0.0725754101135115,
+        -0.7737397204668645
+      ],
+      [
+        -0.21129663115718916,
+        -0.5938629078807577,
+        0.07230313312828618,
+        -0.7729507340347552
+      ],
+      [
+        -0.21135692313333493,
+        -0.5948976701962789,
+        0.07200996261700357,
+        -0.7721655122592219
+      ],
+      [
+        -0.21145660613848902,
+        -0.5959267111897691,
+        0.07169811566017877,
+        -0.771373345936789
+      ]
+    ],
+    "angular_velocities": [
+      [
+        -0.0038859590768814093,
+        -0.0011402570307254794,
+        -0.00038008567690849275
+      ],
+      [
+        -0.004030655455589296,
+        0.0016764286994934076,
+        -0.0013588095664978025
+      ],
+      [
+        -0.0003878096759319298,
+        0.0006000000000000002,
+        -0.0013977835774421693
+      ],
+      [
+        -0.00348649845123291,
+        -0.0003891876935958863,
+        -0.001477030766010285
+      ],
+      [
+        1.785543560981759e-05,
+        -0.00015192399621009824,
+        -0.00038681371212005613
+      ]
+    ],
+    "reference_frame": 1,
+    "constant_frames": [
+      -131340,
+      -131320,
+      -131000
+    ],
+    "constant_rotation": [
+      0.999999669363122,
+      -0.0005776668213350525,
+      0.0005723414110327901,
+      0.0005771560757984276,
+      0.9999994354858921,
+      0.0008921427923870766,
+      -0.000572856449229044,
+      -0.0008918121670889601,
+      0.9999994382531161
+    ]
+  },
   "naif_keywords": {
     "BODY301_RADII": [
       1737.4,
       1737.4,
       1737.4
     ],
-    "BODY_FRAME_CODE": 10020,
+    "BODY_FRAME_CODE": 31001,
     "BODY_CODE": 301,
-    "INS-131335_FOV_BOUNDARY_CORNERS": [
-      0.0005,
-      6.2668,
-      65.4,
-      -0.0125,
-      6.2668,
-      65.4,
-      -0.0125,
-      -6.2392,
-      65.4,
-      0.0005
-    ],
-    "INS-131335_DISTORTION_COEF_X": [
-      2.8912e-19,
-      0.00020899000000000002,
-      4.7727000000000006e-05
-    ],
-    "INS-131335_DISTORTION_COEF_Y": [
-      -1.0119e-18,
-      0.0034982,
-      1.9597e-05
-    ],
-    "INS-131335_BORESIGHT": [
-      -0.006,
-      -0.0187,
-      65.4
-    ],
-    "INS-131335_CENTER": [
-      484.0,
+    "INS-131341_FOV_BOUNDARY_CORNERS": [
+      -1.4914,
+      6.3716,
+      64.9,
+      -1.5314,
+      6.3716,
+      64.9,
```

Comment:
I'm somewhat confused why this file is being edit some rather than a completely new file. Did the isd already exist within ale?

Reply:
There was an existing ISD, but there was no associated driver or test.  I'm not sure where the old ISD came from, so I replaced it with the ISD associated with test data I had been using.